app/assets/javascripts/components/stories/History/HistoryChanges/HistoryChange.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { valueHistory } from '../../../shapes/history';
+
+const HistoryChange = ({
+  oldValue,
+  newValue,
+  title
+}) =>
+  <div className="HistoryChange">
+    <div className="HistoryChange__action" data-id="history-change-title">
+      { title }
+    </div>
+    <div className="HistoryChange__changes">
+      <div data-id="history-old-value"> { oldValue } </div>
+      <div>
+        <i className="mi md-18">arrow_forward</i>
+      </div>
+      <div data-id="history-new-value"> { newValue } </div>
+    </div>
+  </div>
+
+HistoryChange.defaultProps = {
+  oldValue: '---',
```

Comment:
I think these fallbacks are not working. They will only be considered if the value won't actually be passed, but if you call this component with `undefined`/`null` it'll use `undefined`/`null` instead of falling back to the `defaultProps'.

Also, can we hide the changes in ids? Like the "Owned by id", we can hide that since the "Owned by" is already shown. We also don't need to show the change for the name initials since the complete name is already there :smile: 

![Screenshot from 2019-12-11 09-10-22](https://user-images.githubusercontent.com/4325587/70620577-69d9af00-1bf6-11ea-84e1-800f7d5dc05f.png)